src/client/styles/commonStyles.js

```diff
@@ -0,0 +1,97 @@
+import { StyleSheet } from "react-native";
```

Comment:
what are your thoughts on using a utility/component based CSS framework like tailwind or MUI? if we want smth inline we can also use like css-in-js or styledComponents but I personally find it a lot easier to use the former two

Reply:
we can do that for buttons and stuff but we'd still need to style our views and divs and stuff for extra padding/margining or whatever. 

I'm down to utilize a design framework like MUI or antd and combine it w tailwind for cleaner styling though I haven't looked into if tailwind is compatible with react native